chain-tx-enclave-next/enclave-ra/ra-enclave/src/context.rs

```diff
@@ -148,15 +156,20 @@ impl EnclaveRaContext {
     }
 
     /// Creates new certificate params
-    fn create_certificate_params(&self) -> Result<CertificateParams, EnclaveRaContextError> {
+    fn create_certificate_params(
+        &self,
+        expiration_time: Option<Duration>,
+    ) -> Result<CertificateParams, EnclaveRaContextError> {
+        let expiration_time =
+            expiration_time.unwrap_or_else(|| Duration::days(DEFAULT_EXPIRATION_DAYS));
```

Comment:
DEFAULT_EXPIRATION_DAYS  
adding unit to this name or comment about the `unit`?